### ISSUE TITLE
Kernel source for Redmi 9a

### ISSUE THREAD

**Issue by OtakuLinux** [MiCode/Xiaomi_Kernel_OpenSource#1310]
We need kernel source for Xiaomi Redmi 9a

**Comment by BORETS24** [comment]
Yes, I want to buy 9A but kernel sources not found. If kernel sources won't released then I won't to buy so bored device because I'm developer a little and like modify kernel. I found kernel sources for MTK based Note 2 but it was released after almost 4 years passed since dvice presentation! 9A destiny will be the same, wait kernel sources for few years?

**Comment by Flamefusion** [comment]
Xiaomi provide god damn kernel source of 9a
